src/index.html

```diff
@@ -1,13 +1,368 @@
+                >
+              </div>
+
+              <div class="gallery__image-container grid__item--7-12">
+                <img
+                    src="./images//photos/4.png"
+                    alt="gallery image"
+                    class="gallery__image"
+                >
+              </div>
+            </div>
+
+            <div class="gallery__name">
+              Speakers
+            </div>
+          </div>
+
+          <div class="gallery">
+            <div class="gallery__images">
+              <div class="gallery__image-container grid__item--1-4">
+                <img
+                    src="./images//photos/5.png"
+                    alt="gallery image"
+                    class="gallery__image"
+                >
+              </div>
+
+              <div class="gallery__image-container grid__item--5-12">
+                <img
+                    src="./images//photos/6.png"
+                    alt="gallery image"
+                    class="gallery__image"
+                >
+              </div>
+            </div>
+
+            <div class="gallery__name">
+              Audio sunglasses
+            </div>
+          </div>
+
+          <a href="#" class="button">
+              View all
+          </a>
+        </div>
+      </section>
+
+      <section class="page__section" id="testmonials">
+        <div class="container">
+          <div class="page__section-benefits">
+            <h2
+              class="
+                page__section-title
+                grid__item--1-4"
+            >
+              Why buy direct from Bose
+            </h2>
+            <div class="benefits grid__item--7-10">
+              <div class="benefits__subtitles">
+                <div class="benefits__subtitle">
+                  Free 2-day shipping and returns
+                </div>
+                <div class="benefits__subtitle">
+                  90-day risk-free trial
+                </div>
+                <div class="benefits__subtitle">
+                  World class customer service
+                </div>
+                <div class="benefits__subtitle">
+                  My Bose account management
+                </div>
+              </div>
+              <p class="benefits__paragraph">
+                A great product is more than what’s in the box.
+                It’s a promise of premium performance, world-class
+                support, and everything you expect from a trusted brand.
+                It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+              Contact us
+          </h2>
+
+          <div class="contacts__content">
+            <div class="grid__item--1-5">
+              <form action="">
```

Comment:
the form
- must validate inputs
- must NOT reload page on submit
- could reset inputs on submit